models/webhooks/store_url_path_updated.yml

```
@@ -0,0 +1,36 @@
+type: object
+properties:
+  store/url-path/updated:
+    description: |
+      Fires when you edit a product's URL path globally or for a channel locale (i.e. overrides).
+
+      The `context` fields are present for only override updates. 
+    type: object
+    properties:
+      scope:
+        type: string
+      store_id:
+        type: string
+      data:
+        type: object
+        properties:
+          entity_type:
+            type: string
+            enum: product
```

```suggestion
            enum: product
```
Could someone provide the enum values? Is is also `variant` or just `product`?